src/beautifiers/rustfmt.coffee

```diff
@@ -34,7 +34,7 @@ module.exports = class Rustfmt extends Beautifier
     else
       @run(program, ["--version"], help: help)
         .then((stdout) ->
-          if /^0\.(?:[0-4]\.[0-9])/.test(stdout.trim())
+          if /^0\.(?:[0-4]\.[0-9])(?!-nightly)/.test(stdout.trim())
```

Comment:
By using "Executables" you could check version with `rustfmt.isVersion(YOUR_VERSION)`. See https://github.com/Glavin001/atom-beautify/blob/5f968cb9f33ebd235471aaa7ad271c8292fe190e/src/beautifiers/php-cs-fixer.coffee#L75 for example.